bundles/binding/org.openhab.binding.wr3223/src/main/java/org/openhab/binding/wr3223/internal/WR3223Activator.java

```diff
@@ -0,0 +1,27 @@
+package org.openhab.binding.wr3223.internal;
```

Comment:
In this case I guess it's because I flagged what is now line 9 and the change was placed in above that. No worries.